test/collections/Vector2DTest.ooc

```diff
@@ -20,30 +20,107 @@ use ooc-collections
 
 Vector2DTest: class extends Fixture {
 	init: func {
+		sizeXS := 3
+		sizeS := 5
+		sizeM := 10
+		sizeL := 15
+		sizeXL := 20
+		vector2D := Vector2D<Int> new(sizeM, sizeM)
 		super("Vector2D")
-		this add("vector2d list cover create", func {
-			vector2D := Vector2D<Int> new(10, 10)
-			expect(vector2D rowCapacity, is equal to(10))
-			expect(vector2D columnCapacity, is equal to(10))
-
-			for (i in 0 .. 10)
-				for (j in 0 .. 10)
+		this add("capacity", func {
+			expect(vector2D rowCapacity, is equal to(sizeM))
+			expect(vector2D columnCapacity, is equal to(sizeM))
+		})
+		this add("get/set operators", func {
+			for (i in 0 .. vector2D rowCapacity)
+				for (j in 0 .. vector2D columnCapacity)
 					expect(vector2D[i, j], is equal to(0))
-			for (i in 0 .. 10)
-				for (j in 0 .. 10)
+			for (i in 0 .. vector2D rowCapacity)
+				for (j in 0 .. vector2D columnCapacity)
 					vector2D[i, j] = i * j
-			for (i in 0 .. 10)
-				for (j in 0 .. 10)
+			for (i in 0 .. vector2D rowCapacity)
+				for (j in 0 .. vector2D columnCapacity)
 					expect(vector2D[i, j], is equal to(i * j))
 			// Change all values to 10
-			for (i in 0 .. 10)
-				for (j in 0 .. 10)
-					vector2D[i, j] = 10
-			for (i in 0 .. 10)
-				for (j in 0 .. 10)
-					expect(vector2D[i, j], is equal to(10))
-
-			vector2D free()
+			for (i in 0 .. vector2D rowCapacity)
+				for (j in 0 .. vector2D columnCapacity)
+					vector2D[i, j] = sizeM
+			for (i in 0 .. vector2D rowCapacity)
+				for (j in 0 .. vector2D columnCapacity)
+					expect(vector2D[i, j], is equal to(sizeM))
+		})
+		this add("resize increase", func {
+			// Increase array size to 20x20
```

Comment:
It wouldn't be a bad idea to make the test function name more descriptive instead of adding comments to elaborate. When a test fails, the name is printed, but your comments won't be.